### DISCUSSION TITLE
What is the in-browser devtool that the Tailwind Labs youtube channel uses?

### DISCUSSION THREAD

**Discussion by puruzio** [tailwindlabs/tailwindcss#10389] [Help]
I noticed the Tailwind Labs Youtube video shows the presenter using the in-browser inspector to toggle tailwind classes. Does anyone know its name and where to find it?

**Comment by hajirasyafi** [comment]
which video?

**Comment by kfirfitousi** [comment]
I don't know which one they use, but I use this one called "Tailwind CSS Devtools"
Firefox: https://addons.mozilla.org/en-US/firefox/addon/tailwind-css-devtools/
Chrome: https://chrome.google.com/webstore/detail/tailwind-css-devtools/pgamkpjkbfldnmemhcbekimfdnjcgkco
